RedditSharp/Helpers/Helpers.PopulateObject.cs

```diff
@@ -22,5 +25,24 @@ internal static void PopulateObject(JToken json, object obj)
                 jsonSerializer.Populate(reader, obj);
             }
         }
+
+        internal static List<T> PopulateObjects<T>(JToken json, IWebAgent webAgent)
```

Comment:
should this be in `Helpers.PopulateObjects.cs` instead?

Reply:
Yeah probably.

Reply:
It has been moved.